examples/hubert/utils/feature_utils.py

```diff
@@ -114,7 +114,7 @@ def _load_state(model: Module, checkpoint_path: Path) -> Module:
     Returns:
         (Module): The pretrained model.
     """
-    state_dict = torch.load(checkpoint_path)
+    state_dict = torch.load(checkpoint_path, map_location=torch.device("cpu"))
```

Comment:
This function is used as the following, 

```python
model = _load_state(model, checkpoint_path) 
model.to(device) 
```

so moving model first and loading the state dict there seems to be as straight forward.

```python
def _load_state(model: module, checkpoint_path: Path, device=torch.device("cpu")):
    state_dict = torch.load(checkpoint_path, map_location=device)
     ...
```

then change the call site

```python
model.to(device) 
_load_state(model, checkpoint_path, device) 
```